wazuh/config/01-wazuh.sh

```diff
@@ -35,38 +35,54 @@ exec_cmd_stdout() {
 ##############################################################################
 # Check_update
 # This function considers the following cases:
-# - If /var/ossec/etc/ossec-init.conf does not exist -> Action Nothing. There is no data in the EBS. First time deploying Wazuh
-# - If /var/ossec/etc/VERSION does not exist -> Action: Update. The previous version was prior to 3.11.5.
-# - If both files exist: different Wazuh version -> Action: Update. The previous version is older than the current one.
-# - If both files exist: the same Wazuh version -> Acton: Nothing. Same Wazuh version.
+# - If /var/ossec/etc/VERSION does not exist -> Action: Update. Action Nothing. There is no data in the EBS. First time deploying Wazuh
```

Comment:
Why there is 2 actions instead one of them?

Reply:
solved